### DISCUSSION TITLE
APIs publicas

### DISCUSSION THREAD

**Discussion by rafaelprocha** [bacen/pilotord-kit-onboarding#65] [Q&A] [answered]
Caros, duvida conceitual aqui:

Para participação de terceiros, participantes devidamente homologados pelo BCB poderão ter acesso aos Smart Contracts via APIs publicas para executar transações ? Ou esses participantes deverão ser permissionados e terem obrigatóriamente um nó na rede Besu ?

**Comment by aldenio** [comment, marked as answer]
Prezado @rafaelprocha ,

O acesso à infraestrutura do piloto do Drex será concedido unicamente aos participantes homologados pelo BCB. Esses participantes serão permissionados, poderão conectar um nó Besu na rede do Drex e acessar os contratos e serviços pelo seu próprio nó. Não há previsão de publicação de APIs públicas por parte do Banco Central durante o projeto piloto.

Att.
